paraphrase-job.sh

```diff
@@ -33,6 +34,14 @@ else
 fi
 
 make_input_ready(){
+    # remove duplicates before paraphrasing to avoid wasting effort
+    python3 /opt/genienlp/genienlp/data_manipulation_scripts/transform_dataset.py \
+      ${input_dir}/train.tsv \
+      ${input_dir}/train_no_duplicate.tsv \
+      --remove_duplicates \
+      --task ${task_name}
+      cp ${input_dir}/train_no_duplicate.tsv ${input_dir}/train.tsv
```

Comment:
Starting with dataset 18, the dataset will be deduplicated from the beginning, because even when training only with synthetic data deduplication is beneficial.

Reply:
Great. But this is useful for schemaorg datasets as well.

Reply:
Yeah no problem with it.